src/themes/theme.ts

```diff
@@ -45,6 +45,7 @@ export default createTheme({
         borderRadiusLarge: '12px',
         borderRadiusExtraLarge: '20px',
         tableRowHeight: 64,
+        tableRowHeightCompact: 56,
```

Comment:
I'm assuming these are 'px'. Would nice to keep the consistency when setting these values.

Reply:
It's is used to calculate things, so I need `number`